guide/contribute/review.md

```diff
@@ -81,16 +81,20 @@ When you're done with leaving individual feedback, you can submit it. Click on t
 
 #### Best practices when reviewing pull requests
 
-**Do**
+{% include tip/open.html color="green" icon="check" label="Do this" %}
 
-- Leave a suggestion instead of general feedback whenever possible, especially for typo and grammar-related feedback
```

Comment:
I think there's a mix-up here. Some text was removed and other text is duplicated.

Reply:
Yep I did make a mistake here, will change it today. Don't do PRs half asleep